python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,28 @@
+from ..._core import builtin
+from triton._C.libtriton import ir
+
+__all__ = ["buffer_load_to_shared"]
+
+
+@builtin
+def buffer_load_to_shared(dest, ptr, offsets, mask=None, other=None, stride=None, cache_modifier="", _semantic=None):
+    """
+    AMD Buffer load operation. Similar to amdgpu.buffer_load op but directly writes to shared memory instead of into registers.
+
+    Args:
+        dest (shared_memory_descriptor): Destination shared memory descriptor.
+        ptr (tensor): Pointer tensor to load from.
+        offsets (tensor): Offsets tensor for the load operation.
+        mask (tensor, optional): Mask tensor for predicated loads. Defaults to None.
+        other (tensor, optional): Additional tensor for the load operation. Defaults to None.
+        stride (tensor, optional): Stride tensor for the load operation. Defaults to None.
+        cache_modifier (str): Cache modifier specifier. Defaults to "".
+    """
+    builder = _semantic.builder
+
```

Comment:
Can we verify that `offsets`' element type is 32-bit integer? That's the major limitation for buffer load.